tests/wallet/test_transaction_store.py

```diff
@@ -17,14 +17,19 @@
 from chia.wallet.wallet_transaction_store import WalletTransactionStore, filter_ok_mempool_status
 from tests.util.db_connection import DBConnection
 
-coin_1 = Coin(token_bytes(32), token_bytes(32), uint64(12312))
-coin_2 = Coin(token_bytes(32), token_bytes(32), uint64(1234))
-coin_3 = Coin(token_bytes(32), token_bytes(32), uint64(12312 - 1234))
+
+def rand_hash() -> bytes32:
+    return bytes32(token_bytes(32))
```

Comment:
What was the hinting issue here?

Reply:
`bytes` vs `bytes32`. Every call to `token_bytes(32)` needed to pass a `bytes32` object